src/Components/Facility/AddBedForm.tsx

```diff
@@ -150,71 +149,68 @@ export const AddBedForm = (props: BedFormProps) => {
-                  <FieldLabel required id="bedType">
-                    Bed Type
-                  </FieldLabel>
-                  <LegacySelectField
-                    id="bed-type"
-                    fullWidth
-                    name="bed_type"
-                    placeholder=""
-                    variant="outlined"
-                    margin="dense"
-                    options={[
-                      {
-                        id: "",
-                        name: "Select",
-                      },
-                      ...LOCATION_BED_TYPES,
-                    ]}
-                    optionValue="name"
-                    value={bedType}
-                    onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-                      setBedType(e.target.value)
-                    }
-                    errors={errors.bedType}
-                  />
-                </div>
-
-                <div className="flex flex-col gap-3 sm:flex-row sm:justify-between">
-                  <Cancel onClick={handleCancel} />
-                  <Submit onClick={handleSubmit} label={buttonText} />
+      >
+        <div className="mt-10">
+          <Card>
+            <form onSubmit={(e) => handleSubmit(e)}>
+              <div className="md:p-4">
+                <div className="mt-2 grid gap-4 grid-cols-1">
+                  <div>
+                    <TextFormField
+                      name="name"
+                      type="text"
+                      label="Name"
+                      id="name"
+                      required
+                      value={name}
+                      onChange={(e) => setName(e.value)}
+                      error={errors.name}
+                    />
+                  </div>
+                  <div>
+                    <TextAreaFormField
+                      rows={5}
+                      label="Description"
+                      name="description"
+                      value={description}
+                      onChange={(e) => setDescription(e.value)}
+                      error={errors.description}
+                    />
+                  </div>
+                  <div>
+                    <SelectFormField
+                      id="bed-type"
+                      name="bed_type"
+                      placeholder=""
+                      label="Bed Type"
+                      required
+                      options={[
+                        {
+                          id: "",
+                          name: "Select",
+                        },
+                        ...LOCATION_BED_TYPES,
+                      ]}
+                      optionValue={(bedType) => bedType.id}
+                      optionLabel={(bed) => bed.name}
+                      value={bedType}
+                      onChange={({ value }) => {
+                        setBedType(value);
+                      }}
+                      error={errors.bedType}
+                    />
```

Comment:
@print-Sathvik do you have to pass the select as an optional manually? 
The newer components would auttomatically show the `placeholder` as an option based on `reuquired` prop

```suggestion
                    <SelectFormField
                      id="bed-type"
                      name="bed_type"
                      label="Bed Type"
                      required
                      options={LOCATION_BED_TYPES}
                      optionValue={(bedType) => bedType.id}
                      optionLabel={(bed) => bed.name}
                      value={bedType}
                      onChange={({ value }) => {
                        setBedType(value);
                      }}
                      error={errors.bedType}
                    />
```